cmd/influx/pkg.go

```diff
@@ -819,6 +826,83 @@ func (b *cmdPkgBuilder) stackRemoveRunEFn(cmd *cobra.Command, args []string) err
 	return nil
 }
 
+func (b *cmdPkgBuilder) cmdStackUpdate() *cobra.Command {
+	cmd := b.newCmd("update", b.stackUpdateRunEFn, true)
+	cmd.Short = "Update a stack"
+	cmd.Long = `
+	The stack update command updates a stack.
+
+	Examples:
+		# Initialize a stack with a name and description
+		influx stack update -n $STACK_NAME -d $STACK_DESCRIPTION
+
+		# Initialize a stack with a name and urls to associate with stack.
+		influx stack update -n $STACK_NAME -u $PATH_TO_TEMPLATE
```

Comment:
Do these initialize stacks or just update existing stacks? These should also include a stack ID since it's a required parameter.

Reply:
ahhh good point, totally forgot that 🤦